csp_nonce/utils.py

```diff
@@ -1,13 +1,14 @@
 from __future__ import unicode_literals
 import base64
-import nacl.secret
-import nacl.utils
+import os
 
+# Number of random bytes in the nonce:
+NONCE_BYTES = 24
 
 def generate_nonce():
-    """ Return a unique base64 encoded nonce hash """
-    nonce = nacl.utils.random(nacl.secret.SecretBox.NONCE_SIZE)
-    return "{}".format(base64.b64encode(nonce))
+    """ Return a unique base64 encoded nonce hash."""
+    bits = os.urandom(NONCE_BYTES)
+    return base64.b64encode(bits)
```

Comment:
Ah - I think we're gonna need this string formatted.
`return "{}".format(base64.b64encode(bits))` should do it.